src/tests/ftest/pool/attribute.py

```diff
@@ -84,7 +87,7 @@ def verify_list_attr(self, indata, size, buff):
                 "FAIL: Size is not matching for Names in list attr, Expected "
                 "len={0} and received len={1}".format(aggregate_len, size))
         # verify the Attributes names in list_attr retrieve
-        for key in list(indata.keys()):
+        for key in list(indata.values()):
```

Comment:
The comment on line 89 may be wrong here.  The pool attribute list_attr command should be returning a list of pool attribute names.  If we want to verify this output is correct, then we should be checking the dictionary keys (the pool attribute names we provided to the set_attr command):
```suggestion
        for key in indata.keys():
```

Reply:
The structure passed to set_attr is the following:
{key: value}
The attributes of a pool does not follow a key:value pattern, just values:
daos pool list-attrs --pool=$pool
...
...
...
client INFO src/utils/daos.c:151 cmd_args_print()       DAOS system name: daos_server
client INFO src/utils/daos.c:152 cmd_args_print()       pool UUID: 073e27c1-d583-4811-84a7-cbd845c93160
client INFO src/utils/daos.c:153 cmd_args_print()       cont UUID: 00000000-0000-0000-0000-000000000000
client INFO src/utils/daos.c:157 cmd_args_print()       attr: name=NULL, value=NULL
client INFO src/utils/daos.c:161 cmd_args_print()       path=NULL, type=unknown, oclass=UNKNOWN, chunk_size=0
client INFO src/utils/daos.c:167 cmd_args_print()       snapshot: name=NULL, epoch=0, epoch range=NULL (0-0)
client INFO src/utils/daos.c:168 cmd_args_print()       oid: 0.0
Pool attributes:
CCQBNKYTC6B4NZVVP
VO9BWJDMI6HQ8Y32MZPRA0QXZFPYAUOEXLS5LLSXH513XTJ9AND

so pydaos returns a dict in the following form:
{b'0': b'CCQBNKYTC6B4NZVVP', b'1': 'VO9BWJDMI6HQ8Y32MZPRA0QXZFPYAUOEXLS5LLSXH513XTJ9AND'}